content/en/releases/checklist-for-victims-of-a-data-breach.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Checklist for the victims of a data breach"
+description: We collected a comprehensive list of notes especially for the victims of the Vastaamo data breach. If you have been a client of Vastaamo before the year 2018, your confidential information may have been stolen. According to Vastaamo, copies and accessed customer records might exist from early 2019, likely until March 2019.
```

Comment:
We have collected a comprehensive list of information for the victims of the Vastaamo data breach. If you have been a client of Vastaamo your confidential information may have been stolen.

Current information about the data breach can be found at Vastaamo's web pages

Reply:
This is better. I added "before March 2019" because this is not related to recent customer relationships.

Reply:
Careful with the dates...probably best to make it simple for the reader, even if it does mean Vastaamo get more queries.